src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](http://localhost:9000/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_29&issues=AWBUGpoFvB3VRKxIB7gP&open=AWBUGpoFvB3VRKxIB7gP)

[ISSUE_KEY]: AWBUGpoFvB3VRKxIB7gP